helix-core/src/main/java/org/apache/helix/monitoring/mbeans/MonitorDomainNames.java

```diff
@@ -1,33 +0,0 @@
-package org.apache.helix.monitoring.mbeans;
-
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-/**
- * This enum defines all of domain names used with various Helix monitor mbeans.
- */
-public enum MonitorDomainNames {
```

Comment:
Is this a backward compatibility violation?

I think we need a more complete solution to ensure backward compatibility, or we just finish the full migration.

Reply:
The key point is, both classes are packaged in `org.apache.helix.monitoring.mbeans`, so I think it is backward compatible. Although this class is public, client's code doesn't have to change the import. Helix-core.jar depends on helix-common.jar and metrics-common.jar, so it is OK to remove this class in helix-core, as long as we have it in the same package.

Reply:
You are right. I missed the package path.
Since they are the same, it would be fine.